include/splitvector/splitvec.h

```diff
@@ -581,12 +593,12 @@ class SplitVector {
       if (device == -1) {
          SPLIT_CHECK_ERR(split_gpuGetDevice(&device));
       }
-      SPLIT_CHECK_ERR(split_gpuMemPrefetchAsync(_capacity, sizeof(size_t), split_gpuCpuDeviceId, stream));
+      SPLIT_CHECK_ERR(split_gpuMemPrefetchAsync(&_info->capacity, sizeof(size_t), split_gpuCpuDeviceId, stream));
       SPLIT_CHECK_ERR(split_gpuStreamSynchronize(stream));
       SPLIT_CHECK_ERR(split_gpuMemAdvise(_data, capacity() * sizeof(T), advice, device));
-      SPLIT_CHECK_ERR(split_gpuMemAdvise(_size, sizeof(size_t), advice, device));
-      SPLIT_CHECK_ERR(split_gpuMemAdvise(_capacity, sizeof(size_t), advice, device));
-      SPLIT_CHECK_ERR(split_gpuMemPrefetchAsync(_capacity, sizeof(size_t), device, stream));
+      SPLIT_CHECK_ERR(split_gpuMemAdvise(&_info->size, sizeof(size_t), advice, device));
+      SPLIT_CHECK_ERR(split_gpuMemAdvise(&_info->capacity, sizeof(size_t), advice, device));
+      SPLIT_CHECK_ERR(split_gpuMemPrefetchAsync(&_info->capacity, sizeof(size_t), device, stream));
```

Comment:
and prefetch the info

Reply:
This I am not sure how to handle

Reply:
It's good now. I meant, that instead of prefetching or advicing elements of a struct, you want to apply it to the whole struct, and that's what it does now.